vNext/src/OrleansCodeGeneratorBuildMetaPackage/OrleansCodeGeneratorBuildMetaPackage.csproj

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="NuGet">
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.Orleans.OrleansCodeGenerator.Build</PackageId>
+    <Title>Microsoft Orleans Build-time Code Generation</Title>
+    <Description>
+      Microsoft Orleans Build-time Code Generation to install within the grain interfaces and implementation projects.
+    </Description>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IsTool>true</IsTool>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\Orleans.SDK.targets">
+      <Pack>true</Pack>
+      <PackagePath>build\Microsoft.Orleans.OrleansCodeGenerator.Build.targets</PackagePath>
+      <Visible>false</Visible>
+    </None>
+    <None Include="$(OutDir)/ClientGenerator.exe*">
```

Comment:
I am using globbing. The reason for having multiple entries for ClientGenerator this is to not to copy the build generated json files. Later I'd like to address of the "flat output directory" that we are having now.